<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="customplot_Widget" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>351</width>
      <height>261</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad_file"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionChange_color"/>
    <addaction name="actionChoose_data"/>
    <addaction name="actionFunction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionLoad_file"/>
   <addaction name="separator"/>
   <addaction name="actionChange_color"/>
   <addaction name="separator"/>
   <addaction name="actionChoose_data"/>
   <addaction name="separator"/>
   <addaction name="actionFunction"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/menu/save.png</normaloff>:/menu/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionChange_color">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/menu/color.png</normaloff>:/menu/color.png</iconset>
   </property>
   <property name="text">
    <string>Change color</string>
   </property>
  </action>
  <action name="actionChoose_data">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/menu/choose.png</normaloff>:/menu/choose.png</iconset>
   </property>
   <property name="text">
    <string>Choose data set</string>
   </property>
  </action>
  <action name="actionFunction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/menu/function.png</normaloff>:/menu/function.png</iconset>
   </property>
   <property name="text">
    <string>Function</string>
   </property>
  </action>
  <action name="actionLoad_file">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/menu/file_Load.png</normaloff>:/menu/file_Load.png</iconset>
   </property>
   <property name="text">
    <string>Load file</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
